app/src/main/java/io/neurolab/activities/MeditationActivity.java

```diff
@@ -149,24 +153,18 @@ public void onStateChanged(@PlaybackStateCompat.State int state) {
         }
 
         // TODO: Implementation will be carried out in the future when required.
-        public void onPlaybackCompleted(){
+        public void onPlaybackCompleted() {
             Log.i(TAG, "onPlaybackCompleted");
         }
 
         public void onLogUpdated(String message) {
-            if (textDebug != null) {
-                textDebug.append(message);
-                textDebug.append("\n");
-                // Moves the scrollContainer focus to the end.
-                scrollContainer.post(
-                        new Runnable() {
-                            @Override
-                            public void run() {
-                                scrollContainer.fullScroll(ScrollView.FOCUS_DOWN);
-                            }
-                        });
-            }
         }
     }
 
+    @Override
+    public void onBackPressed() {
+        super.onBackPressed();
+        startActivity(new Intent(this, MeditationListActivity.class));
```

Comment:
Why do we need to start an activity here?

Reply:
Will check and revert.

Reply:
Change made. :+1: